aidants_connect_carto_web/templates/places/place_create.html

```diff
@@ -0,0 +1,298 @@
+<script src="{% static 'js/jed.min.js' %}"></script>
+<script src="{% static 'js/yohours.i18n.en.min.js' %}"></script>
+<script src="{% static 'js/yohours.js' %}"></script>
+<script>
+  // leaflet map
+  var map = L.map('map-form').setView([46.7, 2.1], 5);
+  var marker;
+  L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+    attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+  }).addTo(map);
+
+  // jquery typeahead
+  $.typeahead({
+    input: "#id_address_raw",
+    minLength: 5,
+    maxItem: 10,
+    order: "asc",
+    dynamic: true,
+    delay: 500,
+    filter: false,
+    backdrop: {
+      "background-color": "#fff"
+    },
+    emptyTemplate: "Aucune adresse trouvée",
+    display: "properties.label",
+    source: {
+      address: {
+        ajax: function (query) {
+          console.log(query);
+          return {
+            type: "GET",
+            url: "http://localhost:3000/api/address/search?",
+            data: {
+              q: query
+            },
+            path: "features",
+            callback: {
+              done: function (data) {
+                console.log(data);
+                return data;
+              }
+            }
+          }
+        }
+      }
+    },
+    callback: {
+      onInit: function (node) {
+        console.log(node);
+      },
+      onClick: function (node, a, item, event) {
+        populateAddress(item);
+      },
+    }
+  });
+
+  $(".typeahead__button").click(function(event) {
+    event.stopPropagation();
+    clearAddress();
+  });
+
+  function populateAddress(item) {
+    $("#id_address_housenumber").val(item.properties.housenumber);
+    $("#id_address_street").val(item.properties.street);
+    $("#id_address_postcode").val(item.properties.postcode);
+    $("#id_address_citycode").val(item.properties.citycode);
+    $("#id_address_city").val(item.properties.city);
+    $("#id_latitude").val(item.geometry.coordinates[1]);
+    $("#id_longitude").val(item.geometry.coordinates[0]);
+    marker = L.marker(item.geometry.coordinates.reverse()).addTo(map);
+  }
+
+  function clearAddress() {
+    $("#id_address_raw").val("");
+    $("#id_address_housenumber").val("");
+    $("#id_address_street").val("");
+    $("#id_address_postcode").val("");
+    $("#id_address_citycode").val("");
+    $("#id_address_city").val("");
+    $("#id_latitude").val("");
+    $("#id_longitude").val("");
+    map.removeLayer(marker);
+  }
+
+  $('.myOpeningHours').yoHours({
+		bootstrapVersion: 'bootstrap4', // or bootstrap4
+		delay: 700, // in ms, delay before plugin starts parsing input string
+		height: 500, // in pixels, max height of calendar
+    locale: 'en', // must be loaded before yohours like in example above
+    minimal: true,
+    slotDuration: '00:15:00'
+	}, {
+  });
+  $('#yh1_yo-hours-toggle').click(function() {
+    $('#yh1_yo-hours-calendar').slideToggle();
+  });
+</script>
+{% endblock extrajs %}
```

Comment:
Nit: C'est rien du tout, mais la plupart des éditeurs peuvent être configurés pour rajouter une ligne vide à la fin des fichiers, ce qui évite ce warning (`No newline at end of file`) :)

Reply:
ah oui ca m'intéresse :)